docs/pages/eas-workflows/jobs.mdx

```diff
@@ -14,6 +14,15 @@ All jobs support the following properties:
 - `needs` (optional): List of job keys that must complete successfully before this job runs
 - `after` (optional): List of job keys that must complete (success or failure) before this job runs
 
+You can also define an optional [environment](/eas/environment-variables/#environments) property in jobs of type:
+
+- `update`
+- `deploy`
+- `maestro`
+- `custom`
```

Comment:
```suggestion
- `custom-job`
```

I think this should be `custom-job` as per: https://github.com/expo/expo/pull/34540/files#diff-7b6f4a341170791fdff0aea303e3df451e08f0e0187692a2c3f6d3cb2158657aR133.

Reply:
`custom-job` in the linked context is the user-defined key of a job. Think `build_ios`, `e2e_tests`, `setup`.

I think Piotr is listing job types (think `type: build | maestro | ...`). BTW if I remember correctly we've recently decided not to promote `type: custom` and just assume all jobs with no type are custom, so we could remove backticks here and say "custom jobs" in text.